src/Symfony/Bridge/Twig/Extension/ImportMapsExtension.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\ImportMaps\ImportMapManager;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+final class ImportMapsExtension extends AbstractExtension
+{
+    public function __construct(
+        private readonly ImportMapManager $importMapManager,
+    )
+    {
+    }
+
+    public function getFunctions(): iterable
+    {
+        yield new TwigFunction('importmap', [$this, 'importmap'], ['is_safe' => ['html']]);
```

Comment:
Is Twig really compatible with iterable here?

Reply:
I guess yes.

Reply:
yes!

Reply:
The Twig phpdoc says `TwigFunction[]`, which is the notation for an array type. I would not rely on generators working by luck as this could break at any time (and is totally untested in Twig).

Reply:
Btw, Psalm reports it